data/operators/power/substation.json

```diff
@@ -4764,12 +4764,15 @@
         "netz niederösterreich gmbh",
         "netz nö",
         "nö netz",
-        "nö netz ag"
+        "nö netz ag", 
+        "netz niederösterreich",
+        "evn netz"
       ],
-      "note": "Might be related to EVN",
       "tags": {
-        "operator": "Netz Niederösterreich",
-        "power": "substation"
+        "operator": "Netz NÖ",
+        "power": "line"
```

Comment:
This should still be substation

Reply:
True, was a copy/paste mistake

Reply:
I merged as-is and fixed it in 044b00689